docs/sphinx/spelling-wordlist.txt

```diff
@@ -49,6 +50,7 @@ concretizing
 conda
 convecting
 cpp
+dat
```

Comment:
is this needed? I don't see it used?

Reply:
It was giving warning message elsewhere so added it.